src/modules/commons/common.ts

```diff
@@ -27,29 +27,27 @@ export const savingSuccess = <T>(state: ModelState<T>) => {
 	state.error = null
```

Comment:
I am wondering why do we need to refactor this module? Something wrong with the old one?

Reply:
I dont want this module to care about the error response structure. It should only receive and update error || data